src/products/components/ProductList/ProductList.tsx

```diff
@@ -147,7 +147,9 @@ export const ProductList: React.FC<ProductListProps> = props => {
       <ResponsiveTable className={classes.table}>
         <colgroup>
           <col />
-          <col className={classes.colName} />
+          <DisplayColumn column="productName" displayColumns={settings.columns}>
+            <col className={classes.colName} />
+          </DisplayColumn>
```

Comment:
The `DisplayColumn` component is meant to be used when the column can be hidden, product name should always be visible as opposed to the rest of the columns. Additionally, you wrapped the table head column but not the body itself, which would break the table if `productName` is supposed to be hidden.

Reply:
I know, it was discussed already. The fix will be improved.